docs/standard/net-standard.md

```diff
@@ -110,51 +100,66 @@ In order to build .NET Standard libraries in Visual Studio, make sure you have [
 
-- Define APIs that can be used for binary code sharing.
+  **Solution in .NET 5:** When a feature is implemented, it's already available for every .NET 5 app and library because the code base is shared. And since there's no difference between the API specification and its implementation, you're able to take advantage of new features much quicker than with .NET Standard.
 
-Differences:
+- Complex versioning
 
-- .NET Standard is a curated set of APIs, while PCL profiles are defined by intersections of existing platforms.
-- .NET Standard linearly versions, while PCL profiles do not.
-- PCL profiles represents Microsoft platforms while .NET Standard is platform-agnostic.
+  The separation of the API specification from its implementations results in complex mapping between API specification versions and implementation versions. This complexity is evident in the table shown earlier in this article and the instructions for how to interpret it.
 
-### PCL compatibility
+  **Solution in .NET 5:** There's no separation between a .NET 5.x API specification and its implementation. The result is a simplified TFM scheme. There's one TFM prefix for all workloads: `net5.0` is used for libraries, console apps, and web apps. The only variation is a suffix that specifies platform-specific APIs for Windows desktop apps: `net5.0-windows`. Thanks to this TFM naming convention, you can easily tell whether a given app can use a given library. No version number equivalents table like the one for .NET Standard is needed.
 
-.NET Standard is compatible with a subset of PCL profiles. .NET Standard 1.0, 1.1 and 1.2 each overlap with a set of PCL profiles. This overlap was created for two reasons:
+- Platform-unsupported exceptions at run time
 
-- Enable .NET Standard-based PCLs to reference profile-based PCLs.
-- Enable profile-based PCLs to be packaged as .NET Standard-based PCLs.
+  The .NET Standard exposes platform-specific APIs. Your code might compile without errors and appear to be portable to any platform even if it isn't portable. When it runs on a platform that doesn’t have an implementation for a given API, you get runtime errors.
 
-Profile-based PCL compatibility is provided by the [Microsoft.NETCore.Portable.Compatibility](https://www.nuget.org/packages/Microsoft.NETCore.Portable.Compatibility) NuGet package. This dependency is required when referencing NuGet packages that contain profile-based PCLs.
+  **Solution in .NET 5:** The .NET 5 SDK includes analyzers and code fixers that are enabled by default. The platform compatibility analyzer detects unintentional use of APIs that aren't supported on the platforms you intend to run on. For more information, see [Platform compatibility analyzer](analyzers/platform-compat-analyzer.md).
```

Comment:
The code fixers are only available in Visual Studio; not sure if we need to mention that.